### PR TITLE
Updating contributing.md

### PR DESCRIPTION
**Pull request by saroup**
#675 

## What are you trying to address

- Added a bit more clarity in contributing.md about the purpose of this repo and how and what to contribute
- Updated the PR Template so that it references the contributing guide.

## Description of new changes

- Described the purpose of this repo and clarifying that it is not a code repo but should point to good examples of code repos

## For all pull requests
solving part of issue 675

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,28 +17,46 @@ FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
...
 * **Periodic and regular contributions:** \
 If you plan to update the content semi-regularly or regularly, you can be added
 to the project's Contributors group. Please contact [Federica Nocera](https://github.com/fnocera) or [Brock Davis](https://github.com/brockneedscoffee) to be added to the group.
 You will still need to submit a PR against master in order to merge your changes.
 
+## What to contribute
+
+Contribute to the playbook repo anything that is:
+
+* related to the fundamentals
```
Comment (wbreza) [inline CONTRIBUTING.md:43]:
```suggestion
* Related to the fundamentals
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,28 +17,46 @@ FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
...
+* could be publicly visible (avoid confidential info)
```
[the remark this answers is not in the document]
Comment (wbreza) [reply CONTRIBUTING.md:44]:
```suggestion
* Could be publicly visible (avoid confidential info)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -17,28 +17,46 @@ FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
...
+* generally applicable (not specific to internal processes)
+* short code snippets or links to OSS repos (no large code assets)
```
Comment (wbreza) [inline CONTRIBUTING.md:46]:
```suggestion
* Generally applicable (not specific to internal processes)
* Short code snippets or links to OSS repos (no large code assets)
```

**Review by wbreza — APPROVED** [review]
Looks great - just a couple minor suggestions.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -17,28 +17,46 @@ FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
...
+When you have an idea for contribution do one of two things:
+
+* Contact one of the champions in the appropriate fundamental area and discuss your idea with them
+* Look in the opened issues, if you cannot find a similar one then open an issue following the issue templates
+  * **Note:** Adding new labels for an issue are only available for the project's Contributors group. Proposals for new labels are welcomed.
 
-> **Note:** Adding new labels for an issue are only available for the project's Contributors group. Proposals for new labels are welcomed.
+Once you have discussed your contribution:
+
+* Add your changes to a new branch following [this](#branch-naming-convention)
```
[the remark this answers is not in the document]
Comment (saroup) [reply CONTRIBUTING.md:58]: yes I am going to change that

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -17,28 +17,46 @@ FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
...
-contribute some changes, the best way to do it is:
-
-  1. Fork this repository or create a new feature branch
-  2. Add your contributions
-  3. Submit a PR to merge your contribution to this repository
+contribute some changes, the best way to do it is to follow these [steps](#how-to-contribute)
 
 * **Periodic and regular contributions:** \
 If you plan to update the content semi-regularly or regularly, you can be added
 to the project's Contributors group. Please contact [Federica Nocera](https://github.com/fnocera) or [Brock Davis](https://github.com/brockneedscoffee) to be added to the group.
```
[the remark this answers is not in the document]
Comment (saroup) [reply CONTRIBUTING.md:36]: done

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (saroup) [reply CONTRIBUTING.md]: Will make it githubname/workitemid/ since the title can change and the description might not fit in the branch name

**Pull request merged**